src/content/third_party_migration/vendors/nvd.json

```diff
@@ -0,0 +1,21 @@
+{
+    "vendor": "nvd",
+    "description": "National Vulnerability Database",
+    "remotes": [
+        {
+            "url": "https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-{year}.json.gz",
+            "request": "file",
+            "type": "json",
+            "compressed": "gzip",
+            "parameters": {
+                "year": {
+                    "type": "variable-incremental",
+                    "value": [
+                        "2010",
+                        "currentyear"
```

Comment:
- With the objective to make the schema more solid and make the fetcher logic simpler:
    - The same considerations as [here](https://github.com/wazuh/wazuh/pull/14243/files#r928006398), but keyword can be `currentYear`.
    - Rename the type to `variable-incremental-year`